packages/teleport/src/config.ts

```diff
@@ -114,6 +114,10 @@ const cfg = {
     nodeTokenPath: '/v1/enterprise/nodes/token',
     nodeScriptPath: '/scripts/:token/install-node.sh',
     appNodeScriptPath: '/scripts/:token/install-app.sh?name=:name&uri=:uri',
+
+    mfaSignRequestWithTokenPath: '/v1/webapi/u2f/signrequest/token',
+    mfaFetchDevicesPath: '/v1/webapi/mfadevice/:token',
```

Comment:
```suggestion
    mfaDeviceListPath: '/v1/webapi/mfadevice/:tokenId',
```
to keep it consistent elsewhere, i know there is like a few that deviates, but I go with majority